lib/Test/Pod/CoverageChange.pm

```diff
@@ -0,0 +1,246 @@
+package Test::Pod::CoverageChange;
+# ABSTRACT: check perl modules against their pod coverage
+
+=pod
+
+=head1 NAME
+
+Pod coverage calculator test file.
+
+=head1 SYNOPSIS
+
+    use Test::Pod::CoverageChange qw(check);
+
+    Test::Pod::CoverageChange::check('lib', {
```

Comment:
if this is being imported, then it wouldn't need to be fully qualified - but I'd suggest renaming, `check` is way too generic if there are other tests in here. Consider `pod_coverage_ok` or similar.

Reply:
IMHO `check` functionality can be expressed by the package namespace itself. But if your concern is about the conflict problem then I can change the name.

Reply:
BTW your suggested name already in use in this module: https://metacpan.org/pod/Test::Pod::Coverage#pod_coverage_ok(-$module,-%5B$parms,-%5D-$msg-) and we are using it in our module as well.

Reply:
I've changed it to `pod_coverage_syntax_ok`